boot/boot_serial/src/boot_serial.c

```diff
@@ -82,7 +82,7 @@ BOOT_LOG_MODULE_DECLARE(mcuboot);
 #define MCUBOOT_SERIAL_MAX_RECEIVE_SIZE 512
 #endif
 
-#define BOOT_SERIAL_OUT_MAX     (128 * BOOT_IMAGE_NUMBER)
+#define BOOT_SERIAL_OUT_MAX     (168 * BOOT_IMAGE_NUMBER)
```

Comment:
> Mcumgr packets sent over serial are fragmented into frames of 127 bytes or fewer. This 127-byte maximum applies to the entire frame, including header, CRC, and terminating newline.

https://github.com/apache/mynewt-mcumgr/blob/master/transport/smp-console.md#overview

Fix available in https://github.com/mcu-tools/mcuboot/pull/1507